app/assets/stylesheets/application.scss

```diff
@@ -134,7 +144,7 @@ $success-color: #43ac6a;
 $warning-color: #f08a24;
 $info-color: #a0d3e8;
 
-$selected-color: #fa8072;
+$selected-color: #3B3738 ;
```

Comment:
Color `#3B3738` should be written as `#3b3738`<br>Declaration should not have a space before the terminating semicolon